hrms/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -1353,19 +1447,22 @@ def get_taxable_earnings_for_prev_period(self, start_date, end_date, allow_tax_e
 		return (taxable_earnings + opening_taxable_earning) - exempted_amount, exempted_amount
 
 	def get_opening_for(self, field_to_select, start_date, end_date):
-		return (
-			frappe.db.get_value(
-				"Salary Structure Assignment",
-				{
-					"employee": self.employee,
-					"salary_structure": self.salary_structure,
-					"from_date": ["between", [start_date, end_date]],
-					"docstatus": 1,
-				},
-				field_to_select,
-			)
-			or 0
-		)
+		# return (
+		# 	frappe.db.get_value(
+		# 		"Salary Structure Assignment",
+		# 		{
+		# 			"employee": self.employee,
+		# 			"salary_structure": self.salary_structure,
+		# 			"from_date": ["between", [start_date, end_date]],
+		# 			"docstatus": 1,
+		# 		},
+		# 		field_to_select,
+		# 		cache=True,
+		# 	)
+		# 	or 0
+		# )
```

Comment:
Remove commented code? Maybe this method can be removed completely now that salary structure assignment is an internal attribute of the class.